config/routes.rb

```diff
@@ -155,12 +155,8 @@
     get 'generic_works/:id/remove_parent/:parent_id' => 'generic_works#remove_parent'
   end
 
-  if defined?(Sufia::StatsAdmin)
```

Comment:
If you rebase your work, there should be no need for this commit.  It's already done in master.